locals.tf

```diff
@@ -7,6 +7,8 @@ locals {
 
   cluster_name                          = "${var.prefix}-eks"
 
+  specific_azs                          = var.azs == null ? data.aws_availability_zones.available.names : var.azs
```

Comment:
Variable name should be something like `cluster_azs` or `infra_azs` Also given it value is plural is this an array? If it's not the name would be `cluster_az` or `infra_az`

Reply:
This is at all infra level not just for cluster.